src/main/java/org/jabref/logic/plaintextparser/ParserPipeline.java

```diff
@@ -0,0 +1,94 @@
+package org.jabref.logic.plaintextparser;
+
+import org.apache.commons.io.IOUtils;
+import org.apache.http.HttpEntity;
+import org.jabref.Globals;
+import org.jabref.logic.importer.ParseException;
+import org.jabref.logic.importer.fileformat.BibtexParser;
+import org.jabref.logic.net.HttpPostService;
+import org.jabref.model.entry.BibEntry;
+import org.jabref.model.entry.field.StandardField;
+import org.jabref.preferences.JabRefPreferences;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URISyntaxException;
+import java.nio.charset.StandardCharsets;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+/**
+ * This class is used to help making new entries faster by parsing a String.
+ * An external parser called grobid is being used for that.
+ */
+public class ParserPipeline {
```

Comment:
This should be named `GrobitFetcher` or just `Grobit`.

Reply:
It is now called GrobidCitationFetcher and is inside the org.jabref.importer.fetcher package. Moreover, it implements SearchBasedFetcher.java